### ISSUE TITLE
Success message logged as ERR

### ISSUE THREAD

**Issue by ChristianHackAMS** [linux4wilc/driver#55]
Why is this logged as an ERROR level message?

```
2019-07-10T18:15:39.757+10:00 SM01-1751000002 kernel: wilc_sdio mmc1:0001:1 wlan0: ERR [cfg_connect_result:181] Connection Successful: BSSID: ac84c6751fe2
```

**Comment by AdhamAbozaeid** [comment]
yes, this should be an INFO message

**Comment by AdhamAbozaeid** [comment]
Fixed on f9b8665
